lib/Collection.js

```diff
@@ -0,0 +1,197 @@
+/*!
+ * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const axios = require('axios');
+const bedrock = require('bedrock');
+const {EdvClient, EdvDocument} = require('edv-client');
+const https = require('https');
+const instances = require('./instances');
+const {util: {BedrockError}} = bedrock;
+
+const DEFAULT_HEADERS = {Accept: 'application/ld+json, application/json'};
+
+module.exports = class Collection {
+  constructor({type, instance, controllerKey, capability}) {
+    this.type = type;
+    this.instance = instance;
+    this.capability = capability;
+    this.controllerKey = controllerKey;
+  }
+
+  // FIXME: bikeshed name
```

Comment:
Is this still an open question?  What are appropriate alternatives?

Reply:
Yes, it's an open question and I don't know. `get`, `from`, `create` ... it's really a replacement for an async constructor. We can merge without addressing the FIXME, IMO.